haystack/pipeline.py

```diff
@@ -136,6 +141,100 @@ def draw(self, path: Path = Path("pipeline.png")):
         graphviz.layout("dot")
         graphviz.draw(path)
 
+    @classmethod
+    def load_from_yaml(cls, path: Path, pipeline_name: Optional[str] = None):
+        """
+        Load Pipeline from a YAML file defining the individual components and how they're tied together to form
+        a Pipeline. A single YAML can declare multiple Pipelines, in which case an explicit `pipeline_name` must
+        be passed.
+
```

Comment:
Let's add the example yaml from the PR comment here so that it will be shown in our docs.